webapp/channels/src/components/integrations/add_command/add_command.tsx

```diff
@@ -58,7 +58,6 @@ const AddCommand = ({team, actions}: Props) => {
             header={HEADER}
```

Comment:
Although this is unrelated to this PR, is it possible to pass the formatmessage directly here instead of a 't' function?
Eg.
```
header={formatMessage({id: 'intergration.ad............
```

Reply:
So t function is for translation here, and since it is an object and has 3 variants like this, it's better to have them separately.

Reply:
<img width="712" alt="Screenshot 2023-05-08 at 5 58 19 PM" src="https://user-images.githubusercontent.com/43727167/236823805-3920bcca-0d12-41f6-b167-b9ec73e008db.png">